src/urh/signalprocessing/ProtocolSniffer.py

```diff
@@ -211,13 +218,16 @@ def __demodulate_data(self, data):
                                 self.signal.bits_per_symbol, self.signal.center_spacing)
 
         bit_data, pauses, bit_sample_pos = self._ppseq_to_bits(ppseq, samples_per_symbol,
-                                                               self.signal.bits_per_symbol, write_bit_sample_pos=False)
+                                                               self.signal.bits_per_symbol, write_bit_sample_pos=True)
```

Comment:
Why is this change needed?

Reply:
So that `_ppseq_to_bits` populates `bit_sample_pos` array and we are able to calculate more accurate timestamps using bit sample position.